src/node/messages/publish.rs

```diff
@@ -3,7 +3,9 @@ use crate::node::header::Header;
 use crate::node::wire::Wire;
 
 #[derive(Debug)]
-pub struct Publish(pub(crate) BlockHolder);
+pub struct Publish {
+    pub(crate) block_holder: BlockHolder
```

Comment:
Why this change?

Reply:
`publish.0` feels a bit weird, does this make it weirder? It helped me keep in mind I was working with a `BlockHolder` anyways.

Reply:
I think `.0` is ok, although I do think it's a bit ugly. OK let's keep your change and see how it goes.